bom/pom.xml

```diff
@@ -166,6 +181,56 @@
          </dependency>
          <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>infinispan-distexec-demo</artifactId>
+            <version>${project.version}</version>
+         </dependency>
+         <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>infinispan-gridfs-webdav</artifactId>
+            <version>${project.version}</version>
```

Comment:
Why? When someone wants to try the demo they probably want the right version of dependencies...